easybuild/easyconfigs/n/NVHPC/NVHPC-22.7-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,69 @@
+##
+# Author:    Thomas Hayward-Schneider
+##
+name = 'NVHPC'
+version = '22.7'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+# By downloading, you accept the HPC SDK Software License Agreement
+# https://docs.nvidia.com/hpc-sdk/eula/index.html
+# accept_eula = True
+source_urls = ['https://developer.download.nvidia.com/hpc-sdk/%(version)s/']
+local_tarball_tmpl = 'nvhpc_2022_%%(version_major)s%%(version_minor)s_Linux_%s_cuda_multi.tar.gz'
+sources = [local_tarball_tmpl % '%(arch)s']
+checksums = [
+    {
+        local_tarball_tmpl % 'aarch64':
+            '2aae3fbfd2d0d2d09448a36166c42311368f5600c7c346f159c280b412fe924a',
+        local_tarball_tmpl % 'ppc64le':
+            '6dd4fd382c22769e4fa9508714119abd7d1df3dc58c69414a14b0b0dbc34564f',
+        local_tarball_tmpl % 'x86_64':
+            '3ce1c346f8bc7e50defb41c545c8907fdc012ff60b27eb8985cf3213f19d863a',
+    }
+]
+
+local_gccver = '11.2.0'
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.37', '', ('GCCcore', local_gccver)),
+    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
+    ('numactl', '2.0.14', '', ('GCCcore', local_gccver)),
+    ('CUDA', '11.4.1', '', True),
+]
```

Comment:
```suggestion
local_gccver = '11.3.0'
dependencies = [
    ('GCCcore', local_gccver),
    ('binutils', '2.38', '', ('GCCcore', local_gccver)),
    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
    ('numactl', '2.0.14', '', ('GCCcore', local_gccver)),
    ('CUDA', '11.7.0', '', True),
]
```